au/sharing-rules.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Tilt Sharing Rules</title>
+    <script>
+      window.location.href = '/us/sharing-rules.html';
```

Comment:
I've actually been meaning to stop doing this. Github pages supports symlinks so you can just symlink xx/sharing-rules to us/sharing-rules.html. How do you feel about giving this a shot?